independent-projects/arc/tests/src/test/java/io/quarkus/arc/test/contexts/request/propagation/RequestContextPropagationTest.java

```diff
@@ -41,8 +42,9 @@ public void testPropagation() {
         assertNotNull(controller2.getButton());
         assertTrue(controller2.getButton() == controller1.getButton());
 
-        // Store existing instances
-        ContextState state = requestContext.getState();
+        // capture state - two active beans
+        ContextState state1 = requestContext.getState();
```

Comment:
Why `state1` if there are no other states? ;-)

Reply:
Eeh, leftovers from my first attempt at expanding the test. I'll change it, thanks.